### PR TITLE
Draft blog post for version 0.16.2

### PR DESCRIPTION
**Pull request by jbarnoud**

### CODE REVIEW

File: _posts/2017-12-31-release-0.16.2.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Release 0.16.2
+---
+
+Just a few weeks after our last release, we are releasing MDAnalysis version
+0.16.2. The [release notes][notes] list the full details of the changes in this
+version. Like the previous one, this release is dedicated to fixing bugs. It
```
Comment (richardjgowers) [inline _posts/2017-12-31-release-0.16.2.md]: I wouldn't say it's dedicated to bug fixes, the point of it is the deprecations and we need to be loud about this

File: _posts/2017-12-31-release-0.16.2.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+also introduces deprecation warnings for changes that will occur in subsequent
+versions.
+
+The first feature to be deprecated is the [`Timeseries` module][timeseries]
+that will be removed in version 0.17.0. The
+[`MDAnalysis.analysis.base.analysis_class`][analysis_class] function can be
+used to replace the deprecated `Timeseries`.
+
+Quick selectors are also deprecated and will be removed in version 1.0. You can
```
Comment (richardjgowers) [inline _posts/2017-12-31-release-0.16.2.md]: Probably best to give examples of what these are and how to replace them

File: _posts/2017-12-31-release-0.16.2.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+read the discussion in [issue 1377][issue_1377] to learn how to replace the
+different selectors.
+
+Finally, the global [flags][] will disappear in version 1.0 to be replaced with
+more granular keyword arguments in the appropriate methods.
+
+Removing outdated code allows us to keep MDAnalysis robust and consistent. We
```
Comment (richardjgowers) [inline _posts/2017-12-31-release-0.16.2.md]: .. as we prepare for 1.0 where features will become frozen

**Comment by jbarnoud** [comment]
I addressed a couple of comments. But what level of details should there be for the quick selector? We do not want to induce the use of them accidentally by advertising them to much in the blog post that explains there are going to disappear :stuck_out_tongue:

File: _posts/2017-12-31-release-0.16.2.md (hunk 3 above)
Comment (jbarnoud) [inline _posts/2017-12-31-release-0.16.2.md]: The interface is going to be frozen. We will hopefully add features.

**Comment by richardjgowers** [comment]
Ha, I guess, it's just not immediately obvious what is meant by 'quick selectors'.  You could use that feature but not know what we call it.

File: _posts/2017-12-31-release-0.16.2.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Just a few weeks after our last release, we are releasing MDAnalysis version
+0.16.2. The [release notes][notes] list the full details of the changes in this
+version. This version fixes some bugs, but it mostly introduces deprecation
+warnings for changes that will occur in subsequent versions.
+
+The first feature to be deprecated is the [`Timeseries` module][timeseries]
+that will be removed in version 0.17.0. The
+[`MDAnalysis.analysis.base.analysis_class`][analysis_class] function can be
+used to replace the deprecated `Timeseries`.
```
Comment (kain88-de) [inline _posts/2017-12-31-release-0.16.2.md]: a link to the old 0.16.0 blog posts showing the class would be good here.

File: _posts/2017-12-31-release-0.16.2.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Quick selectors are also deprecated and will be removed in version 1.0. You can
+read the discussion in [issue 1377][issue_1377] to learn how to replace the
+different selectors.
+
+Finally, the global [flags][] will disappear in version 1.0 to be replaced with
```
Comment (kain88-de) [inline _posts/2017-12-31-release-0.16.2.md]: https://github.com/MDAnalysis/mdanalysis/issues/782

**Comment by orbeckst** [comment]
I edited the blog post through GH, please have a look, especially @richardjgowers given that you had already approved the prior version.

**Comment by kain88-de** [comment]
The blog looks good to me

**Pull request merged**